v3/src/events/EventDispatcher.js

```diff
@@ -212,7 +212,7 @@ var EventDispatcher = new Class({
     {
         for (var binding in this.bindings)
         {
-            binding.destroy();
+            this.bindings[binding].destroy();
```

Comment:
This doesn't seem right.

[`for ... in`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in) will iterate `this.bindings` properties.

`this.bindings[binding]` would be the proper syntax if we were iterating the keys;
`for (var key in Object.keys(this.bindings))`

Reply:
for...in iterates over the property names/keys.

Reply:
my bad, you are right